.github/workflows/production-daily-desktop.yaml

```diff
@@ -59,6 +59,12 @@ jobs:
 
       - name: Update DB - test coverage
         run: python ./__main__.py --report-type testrail-test-case-coverage --platform desktop --project ALL
+        continue-on-error: true
```

Comment:
I do not like `continue-on-error` in general, but let's be consistent with how the other jobs work. This option may not report the command fails unless we look at the logs.